demisto_sdk/commands/common/tests/docker_test.py

```diff
@@ -486,3 +501,76 @@ def test_get_manifest_from_commit_fails(self, mocker, integration, requests_mock
             with pytest.raises(Exception) as e:
                 DockerImageValidator._get_manifest_from_commit(manifest_url, 'sha1')
             assert str(e.value) == expected
+
+    class TestDeprecatedDockerImage:
+        @pytest.mark.parametrize('image_name, yml_docker_image, is_deprecated_docker, expected', [
+            ('demisto/aiohttp', 'demisto/aiohttp:1.0.2', ('demisto/aiohttp', 'Use the demisto/py3-tools docker image instead.'), False),
+            ('demisto/python', 'demisto/python:1.0.2', '', True)
+        ])
+        def test_deprecated_docker_image(self, integration, requests_mock, image_name, yml_docker_image,
```

Comment:
where do we use the `integration` param?

Reply:
My bad, forgot to remove